include/binary_search_tree.h

```diff
@@ -180,6 +180,10 @@ class BinarySearchTree : public BinaryTree<std::pair<K, V>> {
         }
         return v;
     }
+
+    void remove(Node* u) {
```

Comment:
Why not implement this method?

If we'll do it in the future, let's add a `TODO(allen): xxxx` here.

Reply:
This task has been assigned by @inouetoukyou